Cannot log cache [NIGHTLY]

Reported by nightly users and reproducible for me with 2021.02.07-NB

- Open a cache
- Select log visit

Crash:
```
02-07 12:00:10.718  5539  5539 E AndroidRuntime: FATAL EXCEPTION: main
02-07 12:00:10.718  5539  5539 E AndroidRuntime: Process: cgeo.geocaching, PID: 5539
02-07 12:00:10.718  5539  5539 E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{cgeo.geocaching/cgeo.geocaching.log.LogCacheActivity}: java.lang.ClassCastException: androidx.appcompat.widget.ContentFrameLayout cannot be cast to android.widget.ScrollView
02-07 12:00:10.718  5539  5539 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3763)
02-07 12:00:10.718  5539  5539 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3939)
02-07 12:00:10.718  5539  5539 E AndroidRuntime: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:85)
02-07 12:00:10.718  5539  5539 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
02-07 12:00:10.718  5539  5539 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
02-07 12:00:10.718  5539  5539 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2278)
02-07 12:00:10.718  5539  5539 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
02-07 12:00:10.718  5539  5539 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:246)
02-07 12:00:10.718  5539  5539 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8419)
02-07 12:00:10.718  5539  5539 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
02-07 12:00:10.718  5539  5539 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:596)
02-07 12:00:10.718  5539  5539 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)
02-07 12:00:10.718  5539  5539 E AndroidRuntime: Caused by: java.lang.ClassCastException: androidx.appcompat.widget.ContentFrameLayout cannot be cast to android.widget.ScrollView
02-07 12:00:10.718  5539  5539 E AndroidRuntime: 	at cgeo.geocaching.databinding.LogcacheActivityBinding.bind(LogcacheActivityBinding.java:202)
02-07 12:00:10.718  5539  5539 E AndroidRuntime: 	at cgeo.geocaching.log.LogCacheActivity.onCreate(LogCacheActivity.java:210)
02-07 12:00:10.718  5539  5539 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:8171)
02-07 12:00:10.718  5539  5539 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:8155)
02-07 12:00:10.718  5539  5539 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1309)
02-07 12:00:10.718  5539  5539 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3736)
02-07 12:00:10.718  5539  5539 E AndroidRuntime: 	... 11 more
```

--- a/main/res/layout/logcache_activity.xml
+++ b/main/res/layout/logcache_activity.xml
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/logcache_viewroot"
     android:layout_width="fill_parent"
     android:layout_height="fill_parent"
     android:background="?background_color"

--- a/main/src/cgeo/geocaching/log/LogCacheActivity.java
+++ b/main/src/cgeo/geocaching/log/LogCacheActivity.java
@@ -207,7 +207,7 @@ public class LogCacheActivity extends AbstractLoggingActivity {
     @Override
     public void onCreate(final Bundle savedInstanceState) {
         onCreate(savedInstanceState, R.layout.logcache_activity);
-        binding = LogcacheActivityBinding.bind(getWindow().getDecorView().findViewById(android.R.id.content));
+        binding = LogcacheActivityBinding.bind(findViewById(R.id.logcache_viewroot));
 
         date.init(binding.date, null, getSupportFragmentManager());
         logType.setTextView(binding.type).setDisplayMapper(LogType::getL10n);

